pkg/remote/remote.go

```diff
@@ -63,7 +63,9 @@ USER 0
 ENV PATH="${PATH}:/okteto/bin"
 COPY --from=okteto-cli /usr/local/bin/* /okteto/bin/
 
-
+{{range $key, $val := .OktetoPrefixEnvVars }}
+ENV {{$key}} {{$val}}
+{{end}}
```

Comment:
We are setting several environment variables here, but then, we set some ARG (probably some of them are duplicated). Could there be some side effect? I guess the `ARG` would override the `EVN` defined previously?

Reply:
Yup, I tested and put them at the beginning because of that. Only if the arg is empty it's not overwritten which is something we are not doing.